### DISCUSSION TITLE
Save action UI UX improvement (spinner + toast on save)

### DISCUSSION THREAD

**Discussion by funkyfisch** [home-assistant/frontend#9183] [Automations / scripts / scenes]
As a HA power user that often edits automations, scripts and scenes,

- I would like to have a visual element appear when the save button is clicked (e.g. the save button switch to a spinner instead of the "save" text) that indicates my changes are now being saved on HA, so that I know I really pressed it 

- I would like a visual element to appear when the process of saving is completed (e.g. as a toast notification), showing either the success of the save access or its failure, so that I know that either my changes have indeed been saved and updated, or that I have to try again because it failed.

I can try to create a PR myself for this.

**Comment by capstan1** [comment]
Definitely needed!
If i could add something related here: a indicator is also needed for the automation overview itself. When I hit save on a automation and go back to the verview, all automations reload. The screen goes either blank, all automations show as turned off, or worse: everything seems like it's ready, let's me click anything and then goes blank. This time is rather long on my raspberry pi 3b+.

Have created a automations via blueprints twice several times because I thought something went wrong while it still was loading.

I think a prominent popup stating "automations are being reloaded. Please wait" would help.
